start.cpp

```diff
@@ -7,25 +7,25 @@
 
 //Controller
 class Game{
-private:
     Arena arena;
     Draw draw;
     Snake snake;
     RANDOM random;
 public:
     bool RUN;
     bool TEST;
-    Game(int width, int height, int row, int col, int ori):arena(Arena(width, height)), draw(Draw()),snake(Snake(row,col,ori)), random(RANDOM()), RUN(false), TEST(false) 
+    Game(const int width, const int height,
```

Comment:
Copy constructor of class RANDOM has been deleted because of field "std::random_device rd".Use implicit construction here to avoid compiler reporting a error